app/scripts/services/hpa.js

```diff
@@ -99,6 +100,27 @@ angular.module("openshiftConsole")
     };
 
 
+    var hasV2HPAAnnotations = function(hpaResources) {
+      return _.some(hpaResources, function(hpa) {
+        return $filter('hpaMetrics')(hpa);
+      });
+    };
+
+
+    var hasV2HPAScaler = function(scaleTarget, hpaResources) {
```

Comment:
hasV2HPAScalerWarning to be consistent with the other functions?

Reply:
Unless I'm misunderstanding, we will be filtering this out every time we add it. The only place we want to show the warning is the editor page, but we specifically check `isUnsupportedAPI` there instead of calling the get warnings method. So I think we just avoid adding the warning in `getHPAWarnings`.

Reply:
I can eliminate `hasV2HPAScalerWarning` from `getHPAWarnings`, though then I'll be back to showing this:

![screen shot 2018-02-01 at 3 17 38 pm](https://user-images.githubusercontent.com/280512/35701133-1297d4fa-0763-11e8-8337-17006554760e.png)

Which I'll still need to check for `v2beta1` hpa annotations & filter out anyway.  I'm not sure if I can entirely remove the filtering problem.